include/deal.II/numerics/smoothness_estimator.h

```diff
@@ -458,10 +467,19 @@ namespace SmoothnessEstimator
      * a 5-point Gaussian quarature iterated in each dimension by the maximal
      * wave number, which is the number of modes decreased by one since we start
      * with $k = 0$.
+     *
+     * As the Fourier expansion can only be performed on scalar fields, this
+     * class does not operate on vector-valued finite elements and will
+     * therefore throw an assertion. However, each component of a finite element
+     * field can be treated as a scalar field, respectively, on which Fourier
+     * expansions are again possible. For this purpose, the optional parameter
+     * @p component defines which component of each FiniteElement will be used.
      */
     template <int dim, int spacedim>
     FESeries::Fourier<dim, spacedim>
-    default_fe_series(const hp::FECollection<dim, spacedim> &fe_collection);
+    default_fe_series(
+      const hp::FECollection<dim, spacedim> &fe_collection,
+      const unsigned int component = numbers::invalid_unsigned_int);
```

Comment:
and here

Reply:
I have addressed your comments about the documentation.